docs/deployment/add-ons/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Add-ons"
+description = "Add-on installation guides for Kubeflow on AWS"
```

Comment:
There should be a place to call out add-ons are components which can be installed on top of a Kubeflow deployment

Reply:
we can iterate on this, ok for now.